mathml/relations/html5-tree/css-inline-style-interface.tentative.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>MathML 'ElementCSSInlineStyle` Mixin Tests</title>
+    <link
+      rel="help"
+      href="https://mathml-refresh.github.io/mathml-core/#dom-and-javascript"
+    />
+    <style>
+        math * { background-color: red; }
+    </style>
+    <meta name="assert" content="MathMLElements incorporate a functional ElementCSSInlineStyle interface" />
+    <script src="/mathml/support/mathml-fragments.js"></script>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <span>This tests the presence and functionality of features of the `ElementCSSInlineStyle` interface for MathMLElements</span>
+    <math></math>
+    <script>
+    let mathEl = document.querySelector("math")
+   
+    test(function() {
+      mathEl.style.backgroundColor = "lime"
+        assert_equals(
+          getComputedStyle(mathEl).backgroundColor,
+          "rgb(0, 255, 0)",
+          "The applied background should be green."
+        );
+      }, `The <math> element style property should be present and be functional.`)
+    
+    Object.keys(MathMLFragments).forEach((elName) => {
+      mathEl.innerHTML = MathMLFragments[elName] 
+      
+      test(function() {
+        let el = FragmentHelper.element(mathEl)
+        el.style.backgroundColor = "blue"
+        
```

Comment:
So now you are really testing the correct tags. And for annotation & annotation-xml, el will have "display: none" per the spec. However, that does not seem to cause any problem for the tests? (I guess we can fix the test later if it does).